src/index.html

```diff
@@ -11,7 +11,124 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-applicatio"
+    method="POST"
+    onsubmit="onSubmit()"
+    class="form"
+    >
+      <fieldset class="block-form">
+        <legend>Personal information</legend>
+        <label for="sname" class="first-input">Surname:
+          <input
+          type="text" id="sname"
+          name="sname" autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            id="sname"
            name="sname"
            autocomplete="off"
          >
```
Форматування. кожен атрибут має бути на новому рядку і мають бути правильні відступи